app/models/message.rb

```diff
@@ -41,6 +34,12 @@ class Message < ActiveRecord::Base
 
   after_destroy :notify_deletion_to_subscribers
 
+  def self.default_scope
+    return if User.current_user.try(:staff?)
```

Comment:
Is there a reason you went back to checking roles rather than permissions? @bharat619

Reply:
I guess this is a rebase issue.
Because when I was doing this PR, we were doing it role based. Because of the change in default_scope usage, when I rebased it, it simply thought to remove the `default_scope` block